File: algorithm/src/main/java/baekjoon/p05063/Main.java
package baekjoon.p05063;

import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int t = sc.nextInt();

        while (t-- > 0) {
            System.out.println(
                    solve(sc.nextInt(), sc.nextInt(), sc.nextInt())
            );
        }
    }

    private static String solve(int... input) {
        int diff = input[0] - (input[1] - input[2]);
        if (diff < 0) {
            return "advertise";
        } else if (diff > 0) {
            return "do not advertise";
        } else {
            return "does not matter";
        }
    }

}

File: algorithm/src/main/java/baekjoon/p05063/Main.kt
package baekjoon.p05063

fun main(args: Array<String>) {
    repeat(readLine()!!.toInt()) {
        val (a, b, c) = readLine()!!.split(" ").map { it2 -> it2.toInt() }
        println(solve(a, b, c))
    }
}

private fun solve(vararg input: Int): String {
    val diff = input[0] - (input[1] - input[2])
    return when {
        diff < 0 -> "advertise"
        diff > 0 -> "do not advertise"
        else -> "does not matter"
    }
}